src/int/encoding.rs

```diff
@@ -0,0 +1,51 @@
+//! Const-friendly decoding/encoding operations for [`Int`].
+
+use crate::{Int, Uint};
+
+impl<const LIMBS: usize> Int<LIMBS> {
+    /// Create a new [`Int`] from the provided big endian bytes.
+    ///
+    /// See [`Uint::from_be_slice`] for more details.
+    pub const fn from_be_slice(bytes: &[u8]) -> Self {
+        Self(Uint::from_be_slice(bytes))
+    }
```

Comment:
Should this (and potentially the other methods) handle signed two's complement wire encodings, which are somewhat commonly used? e.g. ASN.1 BER `INTEGER` and SSH `mpint`

Reply:
Sounds like a good idea. Let me look into this to see what this is about/how it works; I have not worked with encodings before.

Reply:
I'd suggest doing that because making `from_be_slice` work with two's complement is trickier than `from_be_hex`, since you have to deal with non-`Word`/`Limb`-aligned sizes.